workflows/mapping/config.yaml

```diff
@@ -1,9 +1,11 @@
 sampletable: pasilla_sampletable.tsv
 sample_dir: pasilla
 assembly: dm6
-data_dir: /data/LCDB/references-test
+data_dir: references-test
 
 shell_prefix: "source lcdb-workflows.bashrc; "
+cluster_default:
+  args: "--time=04:00:00"
```

Comment:
yep, just have to make sure to update the generated template, so if the config changes to:

```yaml
cluster_default:
  mem: '32g'
  lscratch: '100'
```
then the template needs:

```bash
snakemake --cluster-config test-cluster-config.yaml \
    --cluster "sbatch --mem={cluster.mem} --lscratch={cluster.lscratch} --cpus-per-task={threads}"
```